st2common/bin/st2ctl

```diff
@@ -57,16 +57,27 @@ function validate_in_components() {
 
 function st2start() {
   for COM in ${COMPONENTS}; do
-    service ${COM} start
+    service_manager ${COM} start
   done
 }
 
 function st2stop() {
   for COM in ${COMPONENTS}; do
-    service ${COM} stop
+    service_manager ${COM} stop
   done
 }
 
+function service_manager() {
+  if command -v systemctl 2>/dev/null; then
+    systemctl "${2}" "${1}"
```

Comment:
This is not reliable check for systemd!
http://unix.stackexchange.com/questions/196166/how-to-find-out-if-a-system-uses-sysv-upstart-or-systemd-initsystem

> **Detecting systemd**
To check for systemd as the running system manager in the "official" manner, one checks for the existence of /run/systemd/system. This is a directory, in /run, that systemd itself creates at boot, and that other system managers are unlikely to create.

> But that's merely unlikely. This check is already broken, because uselessd creates this directory too.

> Other, unofficial, checks won't work:

> systemd publishes a whole RPC API over D-Bus, which even contains a version name and number; but:
This is not covered by the infamous "Interface Stability Guarantee" and could change tomorrow or at whim.
So too does the lookalike D-Bus server in systemd-shim.
So too does uselessd.
The existence of /run/systemd/private is similarly not guaranteed and similarly duplicated by uselessd.

> https://github.com/StackStorm/st2-packages/blob/master/packages/st2actions/bin/runners.sh#L17